src/CryptManager.cpp

```diff
@@ -80,7 +80,6 @@ static const int KEY_LENGTH = 1024;
  */
 
 static PRNGWrapper *g_pPRNG = nullptr;
-ltc_math_descriptor ltc_mp = ltm_desc;
```

Comment:
I'll need to have another look at this.

Reply:
This statically sets the used math descriptor. See 17.7.19 in [crypt-1.18.2.pdf](https://github.com/libtom/libtomcrypt/releases/download/v1.18.2/crypt-1.18.2.pdf)